vendor/bat-native-ledger/src/bat_publishers.cc

```diff
@@ -35,25 +35,24 @@ BatPublishers::BatPublishers(bat_ledger::LedgerImpl* ledger):
   ledger_(ledger),
   state_(new braveledger_bat_helper::PUBLISHER_STATE_ST),
   server_list_(std::map<std::string, braveledger_bat_helper::SERVER_LIST>()) {
-  calcScoreConsts();
+  calcScoreConsts(state_->min_publisher_duration_);
 }
 
 BatPublishers::~BatPublishers() {
 }
 
-void BatPublishers::calcScoreConsts() {
-  uint64_t min_duration_ms = state_->min_publisher_duration_ * 1000;
-  //TODO: check Warning	C4244	'=': conversion from 'double' to 'unsigned int', possible loss of data
-  a_ = 1.0 / (braveledger_ledger::_d * 2.0) - min_duration_ms;
-  a2_ = a_ * 2;
-  a4_ = a2_ * 2;
+void BatPublishers::calcScoreConsts(const uint64_t& min_duration) {
```

Comment:
side note unrelated to this PR: `calcScoreConsts` doesn't seem like a great name. If we are updating the values they are clearly not constants